test-runner/src/worker.ts

```diff
@@ -16,11 +16,18 @@
 
 import { initializeImageMatcher } from './expect';
 import { TestRunner, fixturePool } from './testRunner';
+import { Console } from 'console';
 
 let closed = false;
 
 sendMessageToParent('ready');
 
+global.console = new Console({
```

Comment:
Will this affect devtools console?

Reply:
No, this shouldn't effect what comes over the protocol.